src/f_table.rs

```diff
@@ -64,9 +64,7 @@ pub fn f_lookup_for_hashmap(opcode_lookup_by_name: HashMap<String, Vec<u8>>) ->
         if idx.len() == 1 {
             let index = idx[0];
             let op = opcode_by_name(name);
-            if op.is_none() {
-                panic!("can't find native operator {:?}", name);
-            }
+            assert!(!op.is_none(), "can't find native operator {:?}", name);
```

Comment:
```suggestion
            assert!(op.is_some(), "can't find native operator {:?}", name);
```
is a tad more readable.